### PR TITLE
Remove old shipment summary worksheet code

### PR DESCRIPTION
**Pull request by mkrump**
## Description

Delete old / unused code relating to shipment summary worksheet.

## Reviewer Notes

@jim it looked like `advance_paperwork.go` was part of the early ssw work and didn't appear to be used currently. but wanted to confirm that it should be part of cleanup.

## Setup

`make server_test`

## Code Review Verification Steps

* [ ] Request review from a member of a different team.
* [ ] Have the Pivotal acceptance criteria been met for this change?

## References

* [Pivotal story](https://www.pivotaltracker.com/n/projects/2181745/stories/164635517)

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -265,7 +265,6 @@ build_tools: bash_version server_deps server_generate build_generate_test_data
 	go build -i -ldflags "$(LDFLAGS)" -o bin/make-dps-user ./cmd/make_dps_user
 	go build -i -ldflags "$(LDFLAGS)" -o bin/make-office-user ./cmd/make_office_user
 	go build -i -ldflags "$(LDFLAGS)" -o bin/make-tsp-user ./cmd/make_tsp_user
-	go build -i -ldflags "$(LDFLAGS)" -o bin/paperwork ./cmd/paperwork
```
Comment (chrisgilmerproj) [inline Makefile]: Do you also need to modify these lines at all? 

https://github.com/transcom/mymove/blob/50ca58972e5952517bcada63563878e3e948e943/Makefile#L218-L221

Comment (mkrump) [reply Makefile]: good point! I actually have't looked at this step before.

I don't think anything needs to be updated here b/c it's only three files in `pkg/paperwork/formtemplates/`  
* form1203template.png
* shipment_summary_worksheet_page1.png
* shipment_summary_worksheet_page2.png

none of which relate to the old stuff.

**Comment by jim** [comment]
@mkrump I think it's OK to remove `advance_paperwork.go` as well. We might reuse some of the code that is called from that file, but nothing actually in it.

**Comment by mkrump** [comment]
@jim I'd already removed `advance_paperwork.go` in this pr, so let me know if you're pointing to another file.

**Pull request merged**